src/Symfony/Bundle/SecurityBundle/DependencyInjection/MainConfiguration.php

```diff
@@ -277,6 +277,13 @@ private function addFirewallsSection(ArrayNodeDefinition $rootNode, array $facto
                     ->scalarNode('role')->defaultValue('ROLE_ALLOWED_TO_SWITCH')->end()
                 ->end()
             ->end()
+            ->arrayNode('session_expiration')
+                ->canBeUnset()
+                ->children()
+                    ->integerNode('max_idle_time')->defaultValue(ini_get('session.gc_maxlifetime'))->min(1)->end()
+                    ->scalarNode('expiration_url')->defaultNull()->end()
```

Comment:
What about `expiration_path`? This seems more consistent with the keys under `form_login`, for example. And this could also be a URL, correct? It looks like you set it up that way :)